skrf/media/media.py

```diff
@@ -1053,15 +1055,15 @@ def shunt_delay_short(self, *args, **kwargs) -> Network:
         """
         return self.shunt(self.delay_short(*args, **kwargs))
 
-    def shunt_capacitor(self, C: NumberLike, *args, **kwargs) -> Network:
+    def shunt_capacitor(self, C: NumberLike, **kwargs) -> Network:
         r"""
         Shunted capacitor.
 
         Parameters
         ----------
         C : number, array-like
             Capacitance in Farads.
-        \*args,\*\*kwargs : arguments, keyword arguments
+        \*\*kwargs : arguments, keyword arguments
             passed to func:`delay_open`
```

Comment:
```suggestion
            passed to func:`capacitor`
```

Reply:
Thanks! I found and corrected quite a few more of those things